bedrock/firefox/templates/firefox/ios.html

```diff
@@ -0,0 +1,146 @@
+        <a href="{{ settings.APPLE_APPSTORE_FIREFOX_LINK }}"><img src="{{ l10n_img('firefox/ios/btn-app-store.svg') }}" alt="{{ _('Download on the App Store') }}"></a>
+      </p>
+      <p class="send-to-device">
+        <button class="send-to button-flat-green">{{ _('Get it now') }}</button>
+      </p>
+    </div>
+
+    <div class="signup">
+      {% set newsletter_title = _('Sign up below for updates and to be notified when it launches.') %}
+      {% set newsletter_id = 'firefox-ios' %}
+      {% set newsletter_details = _('* You’ll only receive updates and announcements about the iOS version of Firefox.') %}
+      {% set newsletter_thankyou_head = _('Thanks for your interest in Firefox for iOS.') %}
+      {% set newsletter_thankyou_content = _('Don’t forget to check your inbox to confirm your request to get updates.') %}
+      {{ email_newsletter_form(newsletter_id, title=newsletter_title, details=newsletter_details, thankyou_head=newsletter_thankyou_head,thankyou_content=newsletter_thankyou_content) }}
+    </div>
+
+    <p class="android-link">
+      <a href="{{ url('firefox.android.index') }}" class="go">{{ _('Get Firefox for Android') }}</a>
+    </p>
+
+  </div>{#-- /.container --#}
+</section>{#-- /#header --#}
+
+<section class="section section-sync" id="sync">
+  <div class="container">
+
+    <h2>{{ _('Sign in to your Firefox Account to access your browsing info on all your mobile devices.') }}</h2>
+
+    <ul class="sync-features">
+      <li class="sync-history">{{ _('History') }}</li>
+      <li class="sync-passwords">{{ _('Passwords') }}</li>
+      <li class="sync-tabs">{{ _('Open tabs') }}</li>
+    </ul>
+
+    <div class="sync-cta">
+
+      <div class="default">
+        <p><a href="{{ url('firefox.sync') }}" class="button-flat">{{ _('Learn more about Sync') }}</a></p>
+      </div>
+
+      <div class="fx-signed-out">
+        <p><button type="button" class="button-flat sync-button">{{ _('Get started with Sync') }}</button></p>
+        <p>{{ _('Already have an account?') }} <a href="https://support.mozilla.org/kb/how-do-i-set-up-firefox-sync" class="go">{{ _('Sign in') }}</a></p>
+      </div>
+
+      <div class="fx-signed-in">
+        <h3>{{ _('You’re already signed in!') }}</h3>
+        <p><a href="{{ url('firefox.sync') }}" class="go">{{ _('Learn more about Sync') }}</a></p>
+      </div>
+
+      <div class="fx-ios-signed-out">
+        <h3>{{ _('Sync your Firefox browsing history on iOS') }}</h3>
+        <p><button type="button" class="button-flat sync-start">{{ _('Get started with Sync') }}</button></p>
+        <p><a href="{{ settings.APPLE_APPSTORE_FIREFOX_LINK }}" class="go">{{ _('Learn more in the App Store') }}</a></p>
+      </div>
+
+    </div>{#-- /.sync-cta --#}
+
+  </div>{#-- /.container --#}
+</section>{#-- /#sync --#}
+
+{% if has_widget %}
+  <div id="send-to-modal-container">
+    {{ send_to_device(platform='ios', title_text=_('Send Firefox to your iOS device')) }}
```

Comment:
We'll be trying an alternate style for iOS, I just haven't done it yet.
![widget-head](https://cloud.githubusercontent.com/assets/205591/9533938/14961362-4cca-11e5-8303-6d2717c299ad.jpg)